lineage/figures/figure11.py

```diff
@@ -0,0 +1,63 @@
+""" This file depicts the distribution of phase lengths versus the states for each concentration. """
+import numpy as np
+import itertools
+import seaborn as sns
+
+from ..Analyze import run_Analyze_over
+from ..data.Lineage_collections import Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM, Lapatinib_Control, Lapt25uM, Lapt50uM, Lap250uM
+from .figureCommon import getSetup, subplotLabel
+
+concs = ["cntrl", "Lapt 25uM", "Lapt 50uM", "Lapt 250uM", "cntrl", "Gem 5uM", "Gem 10uM", "Gem 30uM"]
+data = [Lapatinib_Control, Lapt25uM, Lapt50uM, Lap250uM, Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM]
+
+# Run fitting
+output = run_Analyze_over(data, np.repeat([3, 4], 4))
+lapt_tHMMobj_list = [oo[0] for oo in output[0: 4]]
+lapt_states_list = [oo[1] for oo in output[0: 4]]
+gemc_tHMMobj_list = [oo[0] for oo in output[4: 8]]
+gemc_states_list = [oo[1] for oo in output[4: 8]]
+
+def makeFigure():
+    """ Makes figure 11. """
+
+    ax, f = getSetup((13.2, 6.66), (2, 4))
+    subplotLabel(ax)
+
+    # lapatinib
+    for idx, lapt_tHMMobj in enumerate(lapt_tHMMobj_list): # for each concentration data
+        lpt_g1 = []
+        lpt_g2 = []
+        for lin_indx, lin in enumerate(lapt_tHMMobj.X): # for each lineage list
```

Comment:
Can you make this duplicated code for lapatinib and gemcitabine a single function called twice?

Reply:
Sure. About the parameters, these are dummy values. I wanted to ask you that should I just run the fitting once and print the parameters or should I write a function so in each build it overwrites the parameters?

Reply:
How are these dummy parameters? `run_Analyze_over()` in line 14 runs the fitting.

Reply:
no no, I mean in the table in the methods.

Reply:
You don't need to put the values in the manuscript yet. Just collect them in a separate file so we can look through them. Depending on what they look like we can figure out how to present them.